gorm/README.md

```diff
@@ -1,12 +1,92 @@
-# GORM Package
+# GORM
 
-##### Transaction Management
+This package contains a set of utilities for famous [GORM](http://gorm.io/) library. If used together they can significantly help you to enable persistency in your application.
+
+## Collection Operators
+
+The package provides some helpers which are able to apply collection operators defined in [query](https://github.com/infobloxopen/atlas-app-toolkit/tree/master/query) package to a GORM [query](http://gorm.io/docs/query.html#Query).
+
+### Applying query.Filtering
+
+```golang
+...
+db, assoc, err = gorm.ApplyFiltering(ctx, db, f, &PersonORM{}, &Person{})
```

Comment:
Does it make sense to mention somewhere what is `Person` and `PersonORM` and how they differ?